src/labels/label_point.js

```diff
@@ -50,8 +51,8 @@ export default class LabelPoint extends Label {
     }
 
     updateBBoxes () {
-        let width = (this.size[0] + this.layout.buffer[0] * 2) * this.layout.units_per_pixel * Label.epsilon;
-        let height = (this.size[1] + this.layout.buffer[1] * 2) * this.layout.units_per_pixel * Label.epsilon;
```

Comment:
Right. I realize now the epsilons can be left, because they do not effect the actual size of the texture, only the size of the collision area.